projects/cdk/services/static-request.service.ts

```diff
@@ -23,6 +26,17 @@ export class TuiStaticRequestService {
             return cache;
         }
 
+        if (!url.startsWith(`http`)) {
```

Comment:
Why do we need all this? If it was working under CSR, should we only do this under SSR?